archiso/packages.x86_64

```diff
@@ -108,6 +109,7 @@ usb_modeswitch
 usbmuxd
 usbutils
 vim
+virtualbox-guest-utils-nox
```

Comment:
We don't need this because it's not designed to use with VM, we're targeting T2 Macs